1753683755292-30b3431f487b4cc1863e57a81d78e289.sh

```
@@ -0,0 +1,204 @@
+set -euo pipefail
+
+# ========================
+#       常量定义
+# ========================
+SCRIPT_NAME=$(basename "$0")
+NODE_MIN_VERSION=18
+NODE_INSTALL_VERSION=22
+NVM_VERSION="v0.40.3"
+CLAUDE_PACKAGE="@anthropic-ai/claude-code"
+CONFIG_DIR="$HOME/.claude"
+CONFIG_FILE="$CONFIG_DIR/settings.json"
```

Shellcheck (0.11.0)

[warning] CONFIG_FILE appears unused. Verify use (or export if used externally).

(SC2034)